### PR TITLE
Improve pipeline logs command

### PR DESCRIPTION
**Pull request by pradeepitm12**
Address a few review comments from last PR
Rephrase example message for pipeline log command
Improves output messages if pipeline or runs not found
Verify if the given input pipeline exist
Adds more test cases

Signed-off-by: Pradeep Kumar <pradkuma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/cmd/pipeline/logs.go [hunk 1]
```diff
@@ -129,11 +137,16 @@ func (opts *logOptions) init(args []string) error {
 }
 
 func (opts *logOptions) getAllInputs() error {
-	ps, err := allPipelines(opts.params)
+	ps, err := opts.allPipelines()
```
Comment (vdemeester) [inline pkg/cmd/pipeline/logs.go]: This is not consistent with `verify`. Why this change ?

Comment (pradeepitm12) [reply pkg/cmd/pipeline/logs.go]: `verify()` checks whether the input which is pipeline given by user exist or not.
`getAllInputs()` takes no input from the user and given options to select from.

`tkn pipeline logs foo` showed message no pipelinerun found for the pipeline foo even if the pipeline itself is not there, `verify()` checks if the pipeline exists given by user.

Comment (vdemeester) [reply pkg/cmd/pipeline/logs.go]: my point is on the `go` level. somewhere we do `opts.allPipelines()` and elsewhere we do `verify(opts.pipelineName, opts.param)`. Why not keep `allPipelines(opts.params)` then ?

Comment (pradeepitm12) [reply pkg/cmd/pipeline/logs.go]: Make sense, I will make it consistent and update this.

File: pkg/cmd/pipelinerun/log_reader.go [hunk 2]
```diff
@@ -50,7 +50,7 @@ func (lr *LogReader) Read() (<-chan Log, <-chan error, error) {
 	tkn := lr.Clients.Tekton
 	pr, err := tkn.TektonV1alpha1().PipelineRuns(lr.Ns).Get(lr.Run, metav1.GetOptions{})
 	if err != nil {
-		return nil, nil, fmt.Errorf("%s : %s", msgPRNotFoundErr, err)
+		return nil, nil, fmt.Errorf(err.Error())
```
Comment (vdemeester) [inline pkg/cmd/pipelinerun/log_reader.go]: nit: for the future, we should think of using `golang.org/x/xerrors` to wrap errors :angel:
